src/js/msp/MSPHelper.js

```diff
@@ -699,6 +699,14 @@ MspHelper.prototype.process_data = function(dataHandler) {
                 }
                 break;
 
+            case MSPCodes.MSP_PILOT_NAME:
```

Comment:
See https://github.com/betaflight/betaflight/pull/6924/files#r224974742 for suggestions how this can be done in a better way.

But as stated in the comment, this should not be done now, but only for the next minor release after 4.0.